File: C#/C# 1/5. Conditional-Statements-Homework/Digit-As-Word/DigitAsWord.cs
/* 8. Digit as Word
   Write a program that read a digit (0-9) from the console and shows the digit as a word (in English).
   Print "not a digit" in case of invalid input. Use a switch statement.
   The input will never be an empty line. */

using System;

class DigitAsWord
{
    static void Main()
    {
        string digit = Console.ReadLine();

        switch (digit)
        {
            case "0":
                Console.WriteLine("zero");
                break;
            case "1":
                Console.WriteLine("one");
                break;
            case "2":
                Console.WriteLine("two");
                break;
            case "3":
                Console.WriteLine("three");
                break;
            case "4":
                Console.WriteLine("four");
                break;
            case "5":
                Console.WriteLine("five");
                break;
            case "6":
                Console.WriteLine("six");
                break;
            case "7":
                Console.WriteLine("seven");
                break;
            case "8":
                Console.WriteLine("eight");
                break;
            case "9":
                Console.WriteLine("nine");
                break;
            default:
                Console.WriteLine("not a digit");
                break;
        }
    }
}

File: Front-end/JS Fundamentals/conditional statements/05. Digit as word/digit-as-word.js
function solve(args) {
    var number = +args[0];

    switch (number) {
        case 0: console.log('zero'); break;
        case 1: console.log('one'); break;
        case 2: console.log('two'); break;
        case 3: console.log('three'); break;
        case 4: console.log('four'); break;
        case 5: console.log('five'); break;
        case 6: console.log('six'); break;
        case 7: console.log('seven'); break;
        case 8: console.log('eight'); break;
        case 9: console.log('nine'); break;
        default: console.log('not a digit'); break;
    }
}

solve([2]);
solve([1]);
solve([0]);
solve([5]);
solve([-0.1]);
solve(['hi']);
solve([9]);
solve([10]);